client/pdo/client/commands/collection.py

```diff
@@ -0,0 +1,197 @@
+# Copyright 2024 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import copy
+import logging
+import os
+import toml
+import typing
+
+from zipfile import ZipFile
+
+import pdo.client.builder.shell as pshell
+import pdo.client.builder.script as pscript
+import pdo.client.builder as pbuilder
+from pdo.common.utility import experimental
+
+logger = logging.getLogger(__name__)
+
+__all__ = [
+    'export_contract_collection',
+    'import_contract_collection',
+    'script_command_export',
+    'script_command_import',
+    'do_collection',
+    'load_commands',
+]
+
+# -----------------------------------------------------------------
+# -----------------------------------------------------------------
+def __find_contracts__(context : dict) -> typing.List[str] :
+    """Find all contract save files in a context dictionary
+
+    @type context : dict
+    @param context : the export context
+    """
+    save_files = []
+    for k, v in context.items() :
+        if k == 'save_file' :
+            save_files.append(v)
+        elif isinstance(v, dict):
+            save_files.extend(__find_contracts__(v))
+
+    return save_files
+
+# -----------------------------------------------------------------
+# -----------------------------------------------------------------
+@experimental
+def export_contract_collection(context, context_paths : typing.List[str], contract_cache : str, export_file : str) :
```

Comment:
given that module is called `collection`, why not call this function `export`? This would also make it consistent with the `script_command_export` function? Put differently, would be nice these would be consistent.  (of course this also applies to the import variant ...)

Reply:
good suggestion.

Reply:
going to leave the full name to avoid conflict with builtin import/export functions.